pages/categorie.js

```diff
@@ -13,7 +13,6 @@ export async function getServerSideProps() {
 function Categorie({ lessons }) {
```

Comment:
the page should be called `Category` and the page it resides in `pages/category.js`

Reply:
Deleted